tool/cli.js

```diff
@@ -313,9 +313,24 @@ program
       const { hostname, port } = options;
       let url;
       // Perhaps they typed in a path relative to the content root
-      if (slug.startsWith("files") && slug.endsWith("index.html")) {
+      if (
+        (slug.startsWith("files") || fs.existsSync(slug)) &&
+        (slug.endsWith("index.html") || slug.endsWith("index.md"))
+      ) {
+        if (fs.existsSync(slug) && !CONTENT_TRANSLATED_ROOT) {
```

Comment:
Why do we encourage `CONTENT_TRANSLATED_ROOT` to be set here. Feel a bit arbitrary.

Reply:
Oops. My bad. I had it but accidentally removed it when I refactored. It's suppose to be:
```suggestion
        if (fs.existsSync(slug) && slug.includes('translated-content') && !CONTENT_TRANSLATED_ROOT) {
```